### PR TITLE
Authentication GitHub

### PR DESCRIPTION
**Pull request by BirdyDK**

### CODE REVIEW

File: src/Chirp.Razor/Areas/Identity/Pages/Account/ExternalLogin.cshtml.cs [hunk 1]
```diff
@@ -69,7 +71,8 @@ public async Task<IActionResult> OnGetCallbackAsync(string returnUrl = null, str
                 return RedirectToPage("./Login", new { ReturnUrl = returnUrl });
             }
 
-            var info = await _signInManager.GetExternalLoginInfoAsync();
+            Task<ExternalLoginInfo?> idk = _signInManager.GetExternalLoginInfoAsync();
```
Comment (CptShini) [inline src/Chirp.Razor/Areas/Identity/Pages/Account/ExternalLogin.cshtml.cs:74]: OH GOD, please revert back to the old code, this was just for testing with the debugger :'D

File: src/Chirp.Razor/Areas/Identity/Pages/Account/ExternalLogin.cshtml.cs [hunk 2]
```diff
@@ -55,7 +57,7 @@ public class InputModel
         public IActionResult OnPost(string provider, string returnUrl = null)
         {
             // Request a redirect to the external login provider.
-            var redirectUrl = Url.Page("./ExternalLogin", pageHandler: "Callback", values: new { returnUrl });
+            var redirectUrl = Url.Page("./ExternalLogin", pageHandler: "Callback", values: new { returnUrl });
```
Comment (CptShini) [inline src/Chirp.Razor/Areas/Identity/Pages/Account/ExternalLogin.cshtml.cs:60]: Remove whitespace >:(

**Review by CptShini — CHANGES_REQUESTED** [review]
Seems pretty epic, only have slight changes.

File: src/Chirp.Razor/Areas/Identity/Pages/Account/ExternalLogin.cshtml.cs (hunk 1 above)
Comment (BirdyDK) [reply src/Chirp.Razor/Areas/Identity/Pages/Account/ExternalLogin.cshtml.cs]: I have changed it back to what it was before

File: src/Chirp.Razor/Areas/Identity/Pages/Account/ExternalLogin.cshtml.cs (hunk 2 above)
Comment (BirdyDK) [reply src/Chirp.Razor/Areas/Identity/Pages/Account/ExternalLogin.cshtml.cs]: I have removed the whitespace

**Review by CptShini — APPROVED** [review]

**Review by CptShini — APPROVED** [review]

**Review by Cybermanboss — APPROVED** [review]
Looks good to me

**Pull request merged**